console/console-init/ui/src/graphql-module/queries/Address.ts

```diff
@@ -6,6 +6,7 @@
 import gql from "graphql-tag";
 import { ISortBy } from "@patternfly/react-table";
 import { removeForbiddenChars } from "utils";
+import { getCompplexFilterByPattern, getSimpleFilterByPattern } from "./query";
```

Comment:
There is a typo in getCompplexFilterByPattern. I think we should rename this one as generateComplexFilter and generateSimpleFilter

Reply:
thanks for the suggestion and i make the change with name.
{ generateComplexFilterByPattern, generateSimpleFilterByPattern };